src/parser/Expression.ts

```diff
@@ -348,18 +348,6 @@ export class NamespacedVariableNameExpression extends Expression {
     }
     range: Range;
 
-    // @ts-expect-error override the property
```

Comment:
This comment isn't persisted into the d.ts file, so it causes errors in consumers.